teuthology/exceptions.py

```diff
@@ -51,7 +51,12 @@ class CommandFailedError(Exception):
     Exception thrown on command failure
     """
     def __init__(self, command, exitstatus, node=None, label=None):
-        self.command = command
```

Comment:
From one side this might seem useful for a user to copy and paste command,
but from another point of view, it is bad for debugging, it will be not possible to figure out what
exactly command was before it got to the Exception. Also, the next line joining command with spaces is absolutely not the same as it makes it quoted inside run methods, which makes it hard to reproduce issue.

Reply:
> From one side this might seem useful for a user to copy and paste command,

> but from another point of view, it is bad for debugging, it will be not possible to figure out what
exactly command was before it got to the Exception.

I don't see how would figuring out become difficult since parameter would be printed as it is (at least individually). For example, `['ls', 'dir1']` would become `ls dir1`. I have grepped plenty times and either don't make much of a difference. 

> Also, the next line joining command with spaces is absolutely not the same as it makes it quoted inside run methods, which makes it hard to reproduce issue.

Same as what? 

Umm... are you referring to [this line](https://github.com/ceph/teuthology/blob/master/teuthology/orchestra/run.py#L65) when you say "quoted inside run methods"? I think there's no nice way around that but printing as string definitely takes away effort of deleting a dozen inverted commas every time.

Reply:
Yes, I mean the quotation thing.
Also, if you want to get the exact command, it is already printed in the logs, you just need to find the place where debug is printing the command out before the execution.